src/aics-image-viewer/shared/utils/viewerChannelSettings.ts

```diff
@@ -35,8 +35,13 @@ export interface ViewerChannelSetting {
    * Defaults to false.
    */
   controlPointsEnabled?: boolean;
+  /**
+   * Control points for the transfer function, as histogram bin values.
+   */
   controlPoints?: ControlPoint[];
   ramp?: [number, number];
+  rawRamp?: [number, number];
+  rawControlPoints?: ControlPoint[];
```

Comment:
I've been debating about the best way to handle this. Should I:
1. Keep `controlPoints` and `ramp` as-is, and comment that they are histogram bin indices?
1. Mark `controlPoints` and `ramp` as deprecated, and make a new, separate field that is explicitly for histogram bin indices?

Reply:
Dan: controlPoints2?
Dan: For readability, would prefer to keep `controlPoints` as the main parsed prop. But that is a breaking change.